src/roblox/datatypes/types/cframe.rs

```diff
@@ -264,12 +268,12 @@ impl LuaUserData for CFrame {
         #[rustfmt::skip]
         methods.add_method("GetComponents", |_, this, ()| {
             let pos = this.position();
-            let (rx, ry, rz) = this.orientation();
+            let transposed = this.orientation().transpose();
             Ok((
-                pos.x, pos.y, -pos.z,
-				 rx.x,  rx.y,   rx.z,
-				 ry.x,  ry.y,   ry.z,
-				 rz.x,  rz.y,   rz.z,
+                pos.x, pos.y, pos.z,
+				 transposed.x_axis.x, transposed.x_axis.y,   transposed.x_axis.z,
+				 transposed.y_axis.x, transposed.y_axis.y,   transposed.y_axis.z,
+				 transposed.z_axis.x, transposed.z_axis.y,   transposed.z_axis.z,
```

Comment:
Formatting is a bit funky here now, not sure what would look nicer though, aligning the x/y/z like before or just removing the extra spaces before `transposed_...` 🤔

Reply:
Aligned the positional components with the rest, lemme know what you think

Reply:
Looks good to me!